ffdc/collect_logs.sh

```diff
@@ -0,0 +1,219 @@
+#!/bin/bash
+# IBM_SHIP_PROLOG_BEGIN_TAG
+# *****************************************************************
+#
+# Licensed Materials - Property of IBM
+#
+# (C) Copyright IBM Corp. 2017,2020. All Rights Reserved.
+#
+# US Government Users Restricted Rights - Use, duplication or
+# disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+#
+# *****************************************************************
+# IBM_SHIP_PROLOG_END_TAG
+
+BASEDIR=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+BINDIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+source $BINDIR/logging.sh
+source ${BINDIR}/../prod_env.sh
```

Comment:
Where do these files come from?
I do not see them as part of this PR.
Plus, I don't think we want to have a lot of different `prod_env.sh` scripts. How would user select the correct one for their release?